### DISCUSSION TITLE
Suggestion about Text cursors

### DISCUSSION THREAD

**Discussion by quangtv2702** [antiden/macOS-cursors-for-Windows#49] [General]
I think it would be better to make the white part of the Text cursor a bit thicker, because when I put the mouse pointer on the black background it is a bit hard to see
![image](https://user-images.githubusercontent.com/87218414/175753250-6b8e3f7d-f554-4e0e-bba6-3f48f66b2dc6.png)

**Comment by antiden** [comment]
@vquggg Write the name of the cursor folder you are using.

**Reply by quangtv2702** [reply]
I'm using the Sierra and newer with No shadow currently

**Reply by antiden** [reply]
and size?

**Reply by quangtv2702** [reply]
XtraLarge sir!

**Comment by ricardodrz** [comment]
What exactly I have to do to install these? Can't find  a way to download this files...
